validator/sawtooth_validator/execution/executor.py

```diff
@@ -133,6 +133,15 @@ def _future_done_callback(self, request, result):
 
         elif response.status == processor_pb2.TpProcessResponse.INTERNAL_ERROR:
 
+            LOGGER.exception(
+                "Transaction Processor Internal Error:\n %s",
```

Comment:
Log messages shouldn't have newlines (it makes it more difficult to grep).

Reply:
The log level should also be error, not exception